tracee-rules/main.go

```diff
@@ -168,6 +168,10 @@ func main() {
 				Name:  "rego-enable-parsed-events",
 				Usage: "enables pre parsing of input events to rego prior to evaluation",
 			},
+			&cli.BoolFlag{
+				Name:  "rego-aio-enable",
+				Usage: "enable rego AIO rule for rule indexing and evaluation",
```

Comment:
I think rule indexing is not related to AIO but more to partial evaluation. Here AIO is about compiling all signatures once as a single (huge) module instead of compiling N (smaller) modules.